ui/views.py

```diff
@@ -227,6 +227,10 @@ def dispatch(self, *args, **kwargs):
     def extra_context(self):
         """Add to the context."""
         context = super(RepositoryView, self).extra_context()
+        vocabularies = [
+            x.replace(" ", "_")
```

Comment:
this is the second place where you do this: maybe it's too much, but what about a function that does this replacement (in case you don't need to do only this in the future)